twine/utils.py

```diff
@@ -171,13 +173,14 @@ def get_file_size(filename: str) -> str:
     return f"{file_size:.1f} {size_unit}"
 
 
-def check_status_code(response: requests.Response, verbose: bool) -> None:
+def check_status_code(response: requests.Response, verbose: int) -> None:
     """Generate a helpful message based on the response from the repository.
 
     Raise a custom exception for recognized errors. Otherwise, print the
     response content (based on the verbose option) before re-raising the
     HTTPError.
     """
+    logger = logging.getLogger("LOGGER")
```

Comment:
Again there should be one global `logger = logging.getLogger(__name__)`

Reply:
To clarify, "global" here means at the top of the module, after the imports, same as https://github.com/pypa/twine/pull/669/files#r446512223.